modules/ocf_desktop/manifests/packages.pp

```diff
@@ -19,7 +19,7 @@
       'xarchiver', 'xcape', 'xterm', 'zenmap']:;
     # desktop
     ['desktop-base', 'anacron', 'accountsservice', 'desktop-file-utils',
-      'gnome-icon-theme', 'redshift', 'xfce4-whiskermenu-plugin']:;
+      'gnome-icon-theme', 'redshift', 'xfce4-whiskermenu-plugin', 'arc-theme']:;
```

Comment:
Very convenient that debian already packages this for us.

Reply:
ping

Reply:
Reminder about this

Reply:
Another bump on this, I'd agree sorting this is a good move (however petty of a change it might be :P), and I think `paper-icon-theme` should also be moved here now as I mentioned in an earlier comment.